src/table/settings/economy_settings.ini

```diff
@@ -284,6 +284,14 @@ min      = 800
 max      = 65535
 cat      = SC_EXPERT
 
+[SDT_VAR]
+var      = economy.town_noise_population[3]
+type     = SLE_UINT16
+def      = 400
+min      = 100
+max      = 65535
+cat      = SC_EXPERT
+
```

Comment:
Is this setting required? I think this is a leftover from somewhere else.

Reply:
Yes, it's used here: https://github.com/OpenTTD/OpenTTD/blob/master/src/town.h#L122